src/Xamarin.Android.Build.Tasks/Tests/Xamarin.ProjectTools/Common/Builder.cs

```diff
@@ -302,6 +302,10 @@ protected bool BuildInternal (string projectOrSolution, string target, string []
 				if (Directory.Exists (ndkPath)) {
 					sw.WriteLine (" /p:AndroidNdkDirectory=\"{0}\" ", ndkPath);
 				}
+				string jdkPath = AndroidSdkResolver.GetJavaSdkPath ();
+				if (Directory.Exists (jdkPath)) {
+					sw.WriteLine (" /p:JavaSdkDirectory=\"{0}\" ", jdkPath);
+				}
```

Comment:
There is a change like this in `DotNetCLI.cs`, too:

https://github.com/xamarin/xamarin-android/pull/4567/files#diff-0bf2fe5f45104a6362159a989e222e3bR132

Reply:
merged 4a320361c3cd2f720960ee9955038938d46d088e.

Thanks!